### PR TITLE
fixed unit tests after organisation and craft changes

### PR DESCRIPTION
**Pull request by ho-henry-senior**
**What changes does this PR bring?**

fixed unit tests after organisation and craft changes

**Definition of Done**
The following need to be checked off as they are complete as part of the
PR process. If you are unable to mark off an item as complete, then state
in the comments why this is the case.
- [ ] Have you built the code locally and confirmed its working?
- [ ] Is the build confirmed to be passing on CI?
- [ ] Have you added enough relevant unit tests for your change?
- [ ] Have you added enough relevant E2E tests for your change?
- [ ] Have you updated any relevant documentation as part of this change?
- [ ] Has this change been tested against the Acceptance Criteria?
- [ ] Have you considered how this will be deployed in SIT/Staging/Production?

### CODE REVIEW

**Comment by ho-henry-senior** [comment]
I left some commented code in the unit tests in case there were any more changes to the code and the existing versions were useful

File: src/test/garfile/review/post.controller.test.js [hunk 1]
```diff
@@ -243,7 +243,7 @@ describe('GAR Review Post Controller', () => {
 
       callController().then().then().then()
         .then(() => {
-          expect(res.render).to.have.been.calledWith('app/garfile/review', { cookie });
+          expect(res.render).to.have.been.calledWith('app/garfile/review/index.njk', { cookie });
         });
     });
```
Comment (ame-test) [inline src/test/garfile/review/post.controller.test.js:249]: Nit: Delete commented tests --> refers to code which is not longer needed + version control can get it back:
```javascript
// expect(res.render).to.have.been.calledWith('app/garfile/submit/success/index', { cookie });
```

File: src/test/user/viewDetails/get.controller.test.js [hunk 2]
```diff
@@ -44,15 +44,15 @@ describe('User View Details Get Controller', () => {
...
+    //craftApiStub.rejects('craftApi.getCrafts Example Reject');
     personApiStub.resolves();
 
     const callController = async () => {
       await controller(req, res);
     };
 
     callController().then().then(() => {
-      expect(craftApiStub).to.have.been.calledOnceWithExactly('USER-ID-1');
+      //expect(craftApiStub).to.have.been.calledOnceWithExactly('USER-ID-1');
```
Comment (ame-test) [inline src/test/user/viewDetails/get.controller.test.js:55]: Nit: delete comment code.

**Review by ame-test — APPROVED** [review]
The code is ok. I would prefer the commented code to be delete, but I appreciate it is the choice of the author. 

Note: Unit test should be run during pull request and prevent merge if they fail. It is something that should be implemented.

**Pull request merged**